content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -254,29 +250,26 @@ export class CollapsibleSection extends React.PureComponent {
             </span>
           </h3>
           <div>
-            <button
-              aria-haspopup="true"
-              className="context-menu-button icon"
-              data-l10n-id="newtab-menu-section-tooltip"
-              onClick={this.onMenuButtonClick}
-              ref={this.setContextMenuButtonRef}
-            />
-            {showContextMenu && (
+            <ContextMenuButton
+              tooltip="newtab-menu-section-tooltip"
+              onUpdate={this.onMenuUpdate}
+              refFunction={this.setContextMenuButtonRef}
+            >
               <SectionMenu
                 id={id}
                 extraOptions={extraMenuOptions}
                 eventSource={eventSource}
                 showPrefName={showPrefName}
                 privacyNoticeURL={privacyNoticeURL}
                 collapsed={collapsed}
-                onUpdate={this.onMenuUpdate}
                 isFixed={isFixed}
                 isFirst={isFirst}
                 isLast={isLast}
                 dispatch={dispatch}
                 isWebExtension={isWebExtension}
+                keyboardAccess={this.state.contextMenuKeyboard}
```

Comment:
Is the contextMenuKeyboard state on the parent components still being used? Seems like the prop keyboardAccess prop gets ignored in favor of the ContextMenuButton's state ?

Reply:
no, that should be deleted

Reply:
done!